test/index.html

```diff
@@ -0,0 +1,38 @@
+<!-- index.html -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>June Jaegal, Engagement Project</title>
+    <!-- Add your leaflet and other necessary CSS links here -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
+        integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY="
+        crossorigin=""/>
+    <link rel="stylesheet" href="css/style.css"/>
+
+</head>
+<body>
+    <header>
+        <img id="logo" src="data/phillylogo.png" alt="Logo">
+        <h1>Polling Places in Philadelphia</h1>
+    </header>
+    <main>
+        <div id="map" class="map"></div>
+        <div class="search-and-reviews">
+            <input type="text" id="place-name-filter" placeholder="Search By Place Name">
+            <div id="search-results"></div>
+            <label>
+                <input type="checkbox" id="accessible-checkbox">
+                Accessible Polling Place
+            </label>
+            <div id="review-container"></div>
+            <div id="search-results-container"></div>
```

Comment:
**suggestion (a11y):** The Axe developer tools raised this as an issue. Since you have `overflow-y: auto` in the styles for these elements, they may have a scroll bar when there are more list items than fit on screen. That's fine, as long as the user has a mouse to use. If they're navigating by keyboard however, they'll never be able to get to the element in order to scroll it. To fix this you could say:
```suggestion
            <div id="review-container" tabindex="0"></div>
            <div id="search-results-container" tabindex="0"></div>
```
You can see more information about this issue at https://dequeuniversity.com/rules/axe/4.8/scrollable-region-focusable